lutris/util/linux.py

```diff
@@ -432,7 +444,7 @@ def new_from_ldconfig(cls, ldconfig_line):
     @property
     def arch(self):
         """Return the architecture for a shared library"""
-        detected_arch = ["x86-64", "x32"]
+        detected_arch = ["x86-64", "x32", "libc6"]
```

Comment:
In Exherbo we always cross compile to the "emulated arch target", so it becomes "libc6".